frontend/src/pages/zaaktype/tabs/objecttypen.tsx

```diff
@@ -6,23 +6,39 @@ export const TABS_CONFIG_OBJECTTYPEN: TabConfig<TargetType> = {
   sections: [
     {
       expandFields: [
-        "objecttype",
-        "anderObjecttype",
-        "beginObject",
-        "eindeObject",
+        {
+          name: "objecttype",
+          type: "string",
+          // @ts-expect-error - FIXME: Extending TypedField here.
+          initValue: () => undefined,
```

Comment:
I did this so that the defaults are handled by the backend for now, since I am not sure what the frontend changes will be to make this look to the `_expand.zaakobjecttypen.objecttype` field which contains the options.